tfhe/src/integer/server_key/comparator.rs

```diff
@@ -398,15 +398,16 @@ where {
 
     pub fn unchecked_scalar_sign_parallelized<T>(&self, lhs: &RadixCiphertext, rhs: T) -> Ciphertext
     where
-        T: DecomposableInto<u8>,
+        T: DecomposableInto<u64>,
```

Comment:
I see that change a bit everywhere what is it linked to ?

Reply:
I'm guessing some `<u64>` and `<u8>` got lost in translation but ok understood

Reply:
no issue when mapping the u64 to u8 in some parts of the code ?

Reply:
no because the u64 never exceeds the u8::max

Reply:
ok !